### ISSUE TITLE
Performance hit after switching from an Image Fog scene to one without it

### ISSUE THREAD

**Issue by LosAmosDelCalabozo** [ElfFriend-DnD/foundryvtt-imageFog#20]
This can be reproduced using Foundry 0.7.9 and Image Fog 1.2.0.

Create three scenes:

1) One with normal FOW using:
1.a) Background  Image: "Ravaged Castle [40x50].webp"
1.b) Width: 3920
1.c) Height: 4900

2) Another one with Image Fog using:
2.a) Background  Image: "MermaidInnFloor_Interior.webp"
2.b) Width: 1862
2.c) Height: 1568
2.d) Unexplored Fog Background Image: "MermaidInnFloor_Fog.webp"

3) Another one with normal FOW using:
3.a) Background  Image: "MermaidInnFloor_Interior.webp"
3.b) Width: 1862
3.c) Height: 1568

From a newly loaded game, switching from scene 2 to 1 causes a performance hit, but switching from 3 to 1, or starting in 1 works correctly.

This is noticeable when holding the arrow buttons pressed in an install with no other modules.

This is a video of the issue:

https://www.youtube.com/watch?v=rUfDh4N9HN0

[User Images.zip](https://github.com/ElfFriend-DnD/foundryvtt-imageFog/files/6372776/User.Images.zip)

**Comment by akrigline** [comment]
Performance issues are pretty widespread with this module, I'm afraid I'm not really skilled enough with PIXI to be motivated to try and fix them.

Additionally, with overhead tiles and such coming in core 0.8.x, I'm probably going to let this module die when it breaks.

**Comment by LosAmosDelCalabozo** [comment]
Hi Andrew,

Thanks for getting back to me. Shame, I really like the way this module makes scenes look, I use it a lot for house roofs - I much prefer it to the Roofs & Overhead Tiles module. However seems like 0.8 will do what I need so I'll just wait.

I opened this issue because [the Foundry page for this module](https://foundryvtt.com/packages/image-fog/) says:

> This module should not affect scenes without a fog image set, if you find that it does please raise a Github issue with details.

Which is what is happening here.

**Comment by akrigline** [comment]
Yeah, I appreciate the feedback and the report. Roofs is admittedly not a way I envisioned this being useful but it makes a lot of sense.

**Comment by LosAmosDelCalabozo** [comment]
I really like it cause then the roof shows per room rather than the whole thing, and you can look inside rooms from doors.

Great job anyway, in smaller maps it works like a treat.
